http/v1.md

```diff
@@ -0,0 +1,461 @@
+| X-RateLimit-Reset     | The time at which the current rate limit window resets in UTC epoch seconds. |
+
+For example:
+
+```
+curl -i https://api.example-relayer.com/v1/token_pairs
+HTTP/1.1 200 OK
+Date: Mon, 20 Oct 2017 12:30:06 GMT
+Status: 200 OK
+X-RateLimit-Limit: 60
+X-RateLimit-Remaining: 56
+X-RateLimit-Reset: 1372700873
+```
+When a rate limit is exceeded, a status of **429 Too Many Requests** should be returned.
+
+### Errors
+
+Unless the spec defines otherwise, errors to bad requests should respond with HTTP 4xx or status codes.
+
+#### Common error codes
+
+| Code | Reason                                  |
+| ---- | --------------------------------------- |
+| 400  | Bad Request – Invalid request format    |
+| 404  | Not found                               |
+| 429  | Too many requests - Rate limit exceeded |
+| 500  | Internal Server Error                   |
+| 501  | Not Implemented                         |
+
+#### Error reporting format
+For all **400** responses, see the [error response schema](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/relayer_api_error_response_schema.ts#L1).
+
+```
+{
+    "code": 101,
+    "reason": "Validation failed",
+    "validationErrors": [
+        {
+            "field": "maker",
+            "code": 1002,
+            "reason": "Invalid address"
+        }
+    ]
+}
+```
+
+General error codes:
+
+```
+100 - Validation Failed
+101 - Malformed JSON
+102 - Order submission disabled
+103 - Throttled
+```
+
+Validation error codes:
+
+```
+1000 - Required field
+1001 - Incorrect format
+1002 - Invalid address
+1003 - Address not supported
+1004 - Value out of range
+1005 - Invalid signature or hash
+1006 - Unsupported option
+```
+
+
+### Misc.
+
+*   All requests and responses should be of **application/json** content type
+*   All token amounts are sent in amounts of the smallest level of precision (base units). (e.g if a token has 18 decimal places, selling 1 token would show up as selling `'1000000000000000000'` units by this API).
+*   All addresses are sent as lower-case (non-checksummed) Ethereum addresses with the `0x` prefix.
+
+## REST API
+
+### GET /v1/asset_pairs
+
+Retrieves a list of available asset pairs and the information required to trade them. This endpoint should be [paginated](#pagination).
+
+#### Parameters
+
+*   assetDataA=&assetDataB [string]: these are assetData fields. Returns asset pairs that contain assetDataA and assetDataB (in any order). Setting only assetDataA or assetDataB returns pairs filtered by that asset only (optional)
+
+#### Response
+
+[See response schema](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/relayer_api_token_pairs_response_schema.ts#L1)
+
+```
+[
+    "total": 43,
+    "page": 1,
+    "per_page": 100,
+    "records": [
+        {
+            "assetDataA": {
```

Comment:
Where is the assetData information for A+B? in v0 we also had address in this sturct

Reply:
It's the key of the object so I omitted it.